src/components/MemberListElement.svelte

```diff
@@ -13,14 +13,17 @@
 
 	async function handlePromote() {
 		await promoteChatMember(chatId, memberId);
+		return true;
```

Comment:
Are these needed?

Reply:
no, they are redundant, I tested something and forgot to remove that